### ISSUE TITLE
Personal project getters? 

### ISSUE THREAD

**Issue by tanninettt** [nus-cs2030/2324-s2#232]
Is there a limit to how many times I can use a getter for the personal project?

**Comment by ZHD1987E** [comment]
There’s technically no limit.

But do NOT create too many open doors until the private access modifier become useless.

**Comment by w1nsl** [comment]
I dont think so, if a getter is needed to serve the purpose of the class then it is ok.

**Comment by eleorakow** [comment]
I don't think that there are any limits on the number of getters you can write, but just make sure to only write getters when it is completely necessary to avoid breaking the encapsulation principle

**Comment by euuren** [comment]
I agree, do ensure that the getters are necessary

**Comment by ZHD1987E** [comment]
@tanninettt You might want to close the issue if you find that your queries have been solved. Thanks.
